server/index.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+const cors = require('cors');
+const proxy = require('http-proxy-middleware');
+const ejs = require('ejs');
+
+const port = process.env.PORT || 4000;
+
+const app = express();
+
+app.use(cors());
+app.use(express.static('public'));
+
+app.engine('html', ejs.renderFile);
+app.get('/:id', (req, res) => {
+  res.render('../public/index.html');
+});
+app.set('view engine', 'html');
```

Comment:
Dan and I figured out that we eventually don't need this line of code. (each of us somehow found it by himself)

Reply:
Sounds good. I will test the proxy server without the line.